src/render/SDL_render.c

```diff
@@ -3362,11 +3316,12 @@ static bool RenderPointsWithRects(SDL_Renderer *renderer, const SDL_FPoint *fpoi
         return false;
     }
 
+    const SDL_FPoint scale = { renderer->view->scale.x * renderer->view->logical_scale.x, renderer->view->scale.y * renderer->view->logical_scale.y };
```

Comment:
Should we precalculate this?

Reply:
Yeah, this gets used a bunch, I'll fix that up.